lib/bandit/http1/adapter.ex

```diff
@@ -335,20 +335,26 @@ defmodule Bandit.HTTP1.Adapter do
   response_reasons = %{
     100 => "Continue",
     101 => "Switching Protocols",
+    102 => "Processing",
+    103 => "Early Hints",
     200 => "OK",
     201 => "Created",
     202 => "Accepted",
     203 => "Non-Authoritative Information",
     204 => "No Content",
     205 => "Reset Content",
     206 => "Partial Content",
+    207 => "Multi-Status",
+    208 => "Already Reported",
+    226 => "IM Used",
```

Comment:
Not in 9110

Reply:
[RFC 3229](https://www.rfc-editor.org/rfc/rfc3229#section-10.4.1)